app/celery/reporting_tasks.py

```diff
@@ -19,23 +19,27 @@
 
 @notify_celery.task(name="create-nightly-billing")
 @cronitor("create-nightly-billing")
-def create_nightly_billing(day_start=None):
+def create_nightly_billing(
+    day_start=None,
+    n_days=10,
+    stagger_total_period=timedelta(minutes=5),
```

Comment:
is timedelta serialisable? or will this throw an error when it tries to put it on sqs?

Reply:
This... is a good question.

Reply:
Simpler to just use seconds innit